app/queue.py

```diff
@@ -165,16 +165,15 @@ def expire_inflights(self):
             put_batch_saving_expiry_metric(self.__metrics_logger, self, len(expired))
             current_app.logger.warning(f"Moved inflights {expired} back to inbox {self._inbox}")
 
-    def acknowledge(self, receipt: UUID):
+    def acknowledge(self, receipt: UUID) -> bool:
```

Comment:
Let's add some doc string in there to explain the return please. 🙏

Reply:
With the assumption that stronger typing like this is the way forward, would a typing explanation be better suited in a standards / conventions document rather than in a doc string? Seems a bit odd to have convention/standards documentation embedded in the code. 

That's of course under the assumption that it's an agreed upon direction.

Reply:
I added a simple doc string for the args / return value
```
        """
        Remove the in-flight list from Redis

        Args:
        receipt: UUID
            id of the inflight to remove

        Returns: True if the inflight was found in that queue and removed, False otherwise
        """
```